distributions/demo-server/src/main/resources/config/default_retry.yaml

```diff
@@ -0,0 +1,14 @@
+
+  strategyMappings:
+  - regexes:
+    - "status code: 4\\d\\d"
+    strategy: !<Fatal> {}
+  - regexes:
+    - "simple"
+    strategy: !<Regular>
+      maxAttempts: 5
+      intervalMillis: 100
```

Comment:
I know it is just an example, but in Takeout we have found sub 1 second retries are not that helpful, and often times actively harmful, so we would vote we make the example be 1000 instead of 100